### PR TITLE
Add art assets to AR view

### PR DESCRIPTION
**Pull request by floogulinc**

### CODE REVIEW

File: src/app/play-round/play-round.component.ts [hunk 1]
```diff
@@ -10,40 +12,28 @@ import { MarkerState } from '../ar-view/ar-view.component';
...
-    {name:"rudbeckia hirta", barcodeValue:3, imgPath:"assets/images/1000w-8bit/flowers/rudbeckia hirta.png"},
-    {name:"solidago rigida", barcodeValue:4, imgPath:"assets/images/1000w-8bit/flowers/solidago rigida.png"}
-  ]
+  constructor(public studentRoundService: StudentRoundService, private sessionService: StudentSessionService) { }
 
   ngOnInit() {
-    this.round = new ExampleRound("Test Round", this.testMarkers);
-
-    this.roundReady = true;
+    this.sessionService.joinSession('demo-session');
```
Comment (floogulinc) [inline src/app/play-round/play-round.component.ts:27]: This should removed later when the student can join arbitrary sessions.

**Comment by helloworld12321** [comment]
image

**Comment by helloworld12321** [comment]
image

**Comment by kklamberty** [comment]
What is the best way to test this for review? I have AR Markers printed and from the AR page on the list of test pages, I can see that there are flowers for the AR Markers, but I don't see the bee. Is there something I need to do? I was using the vercel preview on my phone. @floogulinc @helloworld12321

**Comment by floogulinc** [comment]
@kklamberty for the bees you will have to assign one to yourself in Firebase.

**Comment by helloworld12321** [comment]
Right, should have mentioned, sorry :relaxed: the teacher side doesn't set bees yet, so you'll need to use the firebase console, like so: 
image

**Comment by helloworld12321** [comment]
(Jinx! :stuck_out_tongue:)

**Comment by helloworld12321** [comment]
Merging!

**Comment by helloworld12321** [comment]
Steps to get a bee on your phone:

1. On your phone, go to `session-test` and click "anon sign in". Remember the UserID you get. _(You can also click "Join Session", although it's not strictly necessary.)_
2. On a desktop, go to the Firebase console at <https://console.firebase.google.com>, and navigate to Buzz About's database.
3. In the database, go to `/sessions/demo-session/rounds/demo-round/students`, and add a new document with the following contents:
  - Document ID: Your UserID. (Sorry, it's not a lot of fun to type :upside_down_face:)
  - Field: `beeSpecies`
    - Type: `string`
    - Value: `apis_mellifera` (or another bee species ID)
4. Back on your phone, on the `session-test` page, you should see "current bee: honey bee"
4. On your phone, navigate to `ar`. You should see your bee!
5. &hellip;
6. Profit!

**Pull request merged**